readers-frontend/src/components/HeaderNavbar.astro

```diff
@@ -0,0 +1,124 @@
+---
```

Comment:
General: The other ticket has been merged. Can you add the relevant test files for these components? I would stick with the file naming convention `componentname.spec.ts`. Let me know if you need a hand with the tests.